test/features/dataset_schema.feature

```diff
@@ -0,0 +1,43 @@
+@schema_metadata
+Feature: Dataset Schema
+
+    Scenario Outline: Add new dataset metadata fields for default data schema validation
```

Comment:
roll tests together, bdd tests can be run out of order and the later tests which rely on the 'schema' being made above will fail.

alternatively ensure a dataset is made in the 'create-test-data.sh' and use that for verification. while keeping this new dataset creation dynamic with a uuid on its name.

Reply:
Nice to have